test/unit/util/application-logging.test.js

```diff
@@ -94,88 +89,89 @@ tap.test('Application Logging Config Tests', (t) => {
     }
   })
 
-  features.forEach(({ feature, method }) => {
-    t.test(
-      `isApplicationLoggingEnabled should be true when application_logging and ${feature} is truthy`,
-      (t) => {
-        config.application_logging[feature].enabled = true
-        t.equal(loggingUtils.isApplicationLoggingEnabled(config), true)
-        t.end()
-      }
-    )
+  await Promise.all(
+    features.map(async ({ feature, method }) => {
```

Comment:
This one, however, does have the hooks present. Are we sure they are getting applied correctly?

Reply:
it is working.  

```
node test/unit/util/application-logging.test.js | grep 'before test' | wc -l
```

outputs

```
8
```

6 for the fixtures and 2 for the standalone